### PR TITLE
bootstrap migration, part 3

### PR DESCRIPTION
**Pull request by Anpoli13**
add_item page changes
- Change copy
- Reorganise input fields
- Improve UX (changed suggested by @Svyatoslav-Polishchuk)

Preview
- 127 0 0 1_5000_admin_add(iPhone 6_7_8) (2)
- 127 0 0 1_5000_admin_add(Laptop with MDPI screen) (1)

### CODE REVIEW

File: subscribie/blueprints/admin/templates/admin/add_item.html [hunk 1]
```diff
@@ -1,86 +1,146 @@
...
+    </div>                                                                    
+      {% endif %} 
+
+    <div class="container">
+
+      <div class="card mx-auto col-md-6">
+        <form method="post" action="#" enctype="multipart/form-data">
+          {{ form.csrf_token }}
+
+          <div style="display:none">
```
Comment (Svyatoslav-Polishchuk) [inline subscribie/blueprints/admin/templates/admin/add_item.html:39]: @chrisjsimpson is it needed here?

Comment (chrisjsimpson) [reply subscribie/blueprints/admin/templates/admin/add_item.html:39]: @Svyatoslav-Polishchuk The form.csrf_token? Yes I believe so :+1:

Comment (Svyatoslav-Polishchuk) [reply subscribie/blueprints/admin/templates/admin/add_item.html:39]: I meant the hidden div with <input name="email" type="hidden" value='email@example.com' />
@chrisjsimpson

File: subscribie/blueprints/admin/templates/admin/edit.html [hunk 2]
```diff
@@ -1,104 +1,180 @@
...
-    {{ render_field(form.company_name, value=company['name'] ) }}
-    {{ render_field(form.slogan, value=company['slogan'] ) }}
-    <input name="email" type="hidden" value='email@example.com' />
-</fieldset>
-
-<fieldset style="padding:5px">
-<legend>Items</legend>
-    {% for item in items %}
-    <fieldset>
-    <input type="hidden" name="itemIndex" value="{{ loop.index0 }}" />
```
Comment (chrisjsimpson) [inline subscribie/blueprints/admin/templates/admin/edit.html:37]: This is needed, so it's been restored. :+1:

File: subscribie/blueprints/admin/templates/admin/edit.html [hunk 3]
```diff
@@ -1,104 +1,180 @@
...
-    <input type="hidden" name="uuid-{{ loop.index0 }}" value="{{ item.uuid }}" />
```
Comment (chrisjsimpson) [inline subscribie/blueprints/admin/templates/admin/edit.html:38]: This is also needed, so it's been restored. :+1:

File: subscribie/blueprints/admin/templates/admin/add_item.html (hunk 1 above)
Comment (chrisjsimpson) [reply subscribie/blueprints/admin/templates/admin/add_item.html:39]: @Svyatoslav-Polishchuk ah. yes it is at the moment. I need to resolve that at some point.

**Pull request merged**